barretenberg/cpp/src/barretenberg/eccvm/eccvm_flavor.hpp

```diff
@@ -68,7 +68,7 @@ class ECCVMFlavor {
     using Relations = Relations_<FF>;
     using LookupRelation = ECCVMLookupRelation<FF>;
 
-    static constexpr size_t MAX_PARTIAL_RELATION_LENGTH = compute_max_partial_relation_length<Relations, HasZK>();
+    static constexpr size_t MAX_PARTIAL_RELATION_LENGTH = compute_max_partial_relation_length<Relations>();
```

Comment:
don't need this anymore as we are switching to a different evaluation masking technique. same in translator